hardware/raspberrypi/bootmodes/msd.md

```diff
@@ -35,12 +48,35 @@ Check that the output `0x3020000a` is shown. If it is not, then the OTP bit has
 
 If you wish, you can remove the `program_usb_boot_mode` line from config.txt, so that if you put the SD card into another Raspberry Pi, it won't program USB host boot mode. Make sure there is no blank line at the end of config.txt.
 
-## Prepare the USB mass storage device
+You can now boot from an USB Mass Storage device in the same way as booting from SD, see the following paragraph for Raspberry Pi 3B+, CM3+.
+
+<a name="Raspberry Pi 3B+, CM3+"></a>
+## Raspberry Pi 3B+, CM3+
+
+The Raspberry Pi 3B+ and CM3+ support USB Mass Storage boot out of the box. The settings specific to the previous versions of Raspberry Pi do not have to be executed.
+
+This is the same [procedure](../../../installation/installing-images/installation/installing-images/) as for SD cards.
```

Comment:
Did something go wrong with the URL here?

Reply:
Looks like an oopsie, edited.